src/components/Actions/Actions.js

```diff
@@ -44,18 +44,19 @@ export default function Actions() {
               setStartedChallenge(data.startedChallenge);
               setNotifyElectionReminders(data.notifyElectionReminders);
               getChallengerInfo(data.invitedBy);
-              setLoading(false);
+              setLoading(true);
```

Comment:
Remove `setLoading(true)` from here and add at the end of getChallengerInfo function

Need to wait for the getChallengerInfo function to execute before loading data when nothing is in localStorage, otherwise this issue comes up because we are trying to load data that hasn't been populated yet
![image](https://user-images.githubusercontent.com/66650721/171069492-cb2873c9-222d-4f17-8c8f-68f52696e4ec.png)